cinema/views.py

```diff
@@ -0,0 +1,53 @@
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from cinema.models import Movie
+from cinema.serializer import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list_view(request: HttpRequest) -> HttpResponse:
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    if request.method == "POST":
+        serializer = MovieSerializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
+            return Response(
+                serializer.data, status=status.HTTP_201_CREATED
+            )
+        return Response(
+            serializer.errors, status=status.HTTP_400_BAD_REQUEST
+        )
+
+
+@api_view(["GET", "PUT", "DELETE"])
+def movie_detail_view(
+        request: HttpRequest, pk: int
+) -> HttpResponse:
+    movie = get_object_or_404(Movie, pk=pk)
+    if request.method == "GET":
+        serializer = MovieSerializer(movie)
+        return Response(
+            serializer.data, status=status.HTTP_200_OK
+        )
+    elif request.method == "PUT":
+        serializer = MovieSerializer(
+            movie, data=request.data
+        )
+        if serializer.is_valid():
+            serializer.save()
+            return Response(
+                serializer.data, status=status.HTTP_204_NO_CONTENT

```

Comment:
the last one)
if you PUT or PATCH something, the response status code should be 200 OK